src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -130,7 +130,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Target Name="_CheckForNETCoreSdkIsPreview"
```

Comment:
@rainersigwald is there any perf gain with having both of the conditions on the target instead of the task in this case?

Reply:
Presumably some, since we wouldn't have to even think about the task in that case, but I can't imagine it's significant.

Reply:
Gotta get that perf. Will lift condition to the target instead